app/controllers/GoldenTicketController.scala

```diff
@@ -188,8 +205,12 @@ object GoldenTicketController extends SecureCFPController {
       scala.concurrent.Future {
         Proposal.findById(proposalId) match {
           case Some(proposal) =>
+            val proposalIdsWithDelayedReview = ReviewByGoldenTicket.delayedReviewsReasons(uuid).keySet
```

Comment:
Performance : here you might want to use a lighter Redis call such as Redis HKEYS directly. The HGETALL is a O(N) slow operation (+ the JSON Parsing). 
What about `ReviewByGoldenTicket.allProposalIdsForUser(uuid)` that would return a Set[ProposalID] (or a Set[String])

Reply:
no parsing involved in the `ReviewGT:DelayedReviews:ByEventCode:DevoxxFR2022:AndAuthor:xxxx` collection : this is a simple `proposalId (String) -> reason (String)` map

But I agree with you on introducing a `allProposalIdsHavingDelayedReviewsForUser(uuid)` in both `Review` & `ReviewByGoldenTicket` models
fixed it through 42055757

Reply:
thanks